mirage/views/node.ts

```diff
@@ -0,0 +1,14 @@
+import { HandlerContext, Schema } from 'ember-cli-mirage';
+
+export function createNode(this: HandlerContext, schema: Schema) {
+    const attrs = this.normalizedRequestAttrs();
+    const node = schema.nodes.create(attrs);
+    const now = (new Date()).toISOString();
+    node.attrs.dateModified = now;
+    node.attrs.dateCreated = now;
+    const userId = schema.roots.first().attrs.currentUserId;
```

Comment:
I don't think you need to do this via `attrs`; `schema.roots.first().currentUserId` should be fine